.github/workflows/ci.yaml

```diff
@@ -45,10 +57,11 @@ jobs:
         if: matrix.python-version == '3.8'
         with:
           env_vars: PYTHON
-          fail_ci_if_error: true
+          # TODO: re-enable errors when rate limit is resolved?
+          # fail_ci_if_error: true
```

Comment:
Technically this is [still throwing an error uploading the coverage](https://github.com/secondlife/python-llbase/actions/runs/13189230868/job/36818543561#step:8:40) since we can only upload one batch of coverage data every hour or something? Until we can resolve that, let's not block changes on whether Codecov accepts our coverage data.

```text
[2025-02-06T22:38:27.615Z] ['error'] There was an error running the uploader: Error uploading to [https://codecov.io:](https://codecov.io/) Error: There was an error fetching the storage URL during POST: 429 - {"message":"Rate limit reached. Please upload with the Codecov repository upload token to resolve issue. Expected time to availability: 2427s."}
```